sdk/python/kubeflow/training/constants/constants.py

```diff
@@ -22,12 +26,25 @@
 
 TFJOB_LOGLEVEL = os.environ.get('TFJOB_LOGLEVEL', 'INFO').upper()
 
-# How long to wait in seconds for requests to the ApiServer
-APISERVER_TIMEOUT = 120
-
-# TFJob Labels Name
+# TFJob Label Names
 TFJOB_GROUP_LABEL = 'group-name'
 TFJOB_NAME_LABEL = 'job-name'
 TFJOB_TYPE_LABEL = 'replica-type'
 TFJOB_INDEX_LABEL = 'replica-index'
 TFJOB_ROLE_LABEL = 'job-role'
+
+# PyTorchJob K8S constants
+PYTORCHJOB_GROUP = 'kubeflow.org'
+PYTORCHJOB_KIND = 'PyTorchJob'
+PYTORCHJOB_PLURAL = 'pytorchjobs'
+PYTORCHJOB_VERSION = os.environ.get('PYTORCHJOB_VERSION', 'v1')
+
+PYTORCH_LOGLEVEL = os.environ.get('PYTORCHJOB_LOGLEVEL', 'INFO').upper()
+
+# PyTorchJob Label Names
+PYTORCHJOB_CONTROLLER_LABEL = 'controller-name'
```

Comment:
Other framework's constants are not populated?

Reply:
This file is added by ourselves for e2e test cases? I notice mxnet and xgboost are missing. If we don't have enough time to write test case for other frameworks, it's totally fine. Let's leave a TODO there?

Reply:
It's also used in client methods: 
https://github.com/mesosphere/tf-operator/blob/75734f854a5b35715f068327d69de6185396b5d0/sdk/python/kubeflow/training/api/tf_job_client.py#L122-L125
I will add a comment, sure. The reason why I didn't add similar constants for `mxnet` and `xgboost` is that there are no clients for these frameworks currently implemented. Adding support for two more frameworks to the SDK is out of scope for this PR and should be addressed separately IMO.